### ISSUE TITLE
`pipx install https://github.com/.../master.zip` fails

### ISSUE THREAD

**Issue by BenjaminRodenberg** [precice/config-visualizer#14]
I just tried to install the config visualizer following the instructions in the `README.md` and ran into the following problem:
```
$ pipx install https://github.com/precice/config-visualizer/archive/master.zip
Package cannot be a url
```

I guess my `pipx` version is just too old: 
```
$ pipx --version
0.12.3.1
```
I think it would be useful to mention the minimum required `pipx` version in the `README.md` (if this is the reason).

I'm currently upgrading `pipx` and will comment here when I know more.

**Comment by BenjaminRodenberg** [comment]
Upgrading `pipx` turned out to be much more complicated than expected. I now worked around the problem by using `pip3`:

```
pip3 install --user https://github.com/precice/config-visualizer/archive/master.zip
```

**Comment by fsimonis** [comment]
I intent to publish releases of the config visualizer on pypi. Then this becomes way more straight forward.

**Comment by fsimonis** [comment]
An alternative way of installing it is to clone the repository, `cd` into it and run `pipx install .`

**Comment by fsimonis** [comment]
I published the current version as 0.2.0 on PyPi.

```
pipx install precice-config-visualizer
```
